style.css

```diff
@@ -0,0 +1,50 @@
+
```

Comment:
Please try to think how can be done this
![image](https://user-images.githubusercontent.com/104353194/168749455-3e8b09fd-56b4-4cdf-9347-cbbe59e220fa.png)